### ISSUE TITLE
Specs

### ISSUE THREAD

**Issue by jgallen23** [firstandthird/hapi-trailing-slash#1]
Hapi plugin to make `/url` redirect to `/url/` and other way around.  Basically if I define `/user` and somebody types in `/user/` it will redirect to the right url.

**Comment by dawnerd** [comment]
https://github.com/hapijs/hapi/blob/master/API.md#new-serveroptions `stripTrailingSlash` ;)

**Comment by jgallen23** [comment]
yeah, but that doesn't redirect. it just strips so now you have two urls that go to the same thing

**Comment by dawnerd** [comment]
Oh, I could have sworn it did a redirect. TIL

**Comment by jgallen23** [comment]
@orthagonal can you check that first?
